content/W2017/W17-event-dinner-with-professors.md

```diff
@@ -0,0 +1,27 @@
+title: Dinner with the Professors
+Date: 2017-03-1
+Category: Events
+Tags: events, social
+Slug: dinner-with-professors-W17
+Author: Talia McCormick
+Summary: Join WiCS for our second Dinner with the Professors event!
+
+Come have dinner with the brilliant women who are professors at Waterloo! This
+is a wonderful opportunity to learn about their research, academia, teaching,
+and your own academic interests. 
+
+Note: Tickets for students are sold for $10 in MathSoc, until Wednesday, March
+1. Registration for professors and for grad students who are interested in
```

Comment:
It looks fine when I run locally.
![screenshot 2017-03-02 22 36 39](https://cloud.githubusercontent.com/assets/13280845/23537245/c530782a-ff98-11e6-9b21-2741597d128c.png)
Putting the "1." back a line will make the previous line greater than 80 characters wide. I could change it to say "March first" instead. Alternatively, I could move March onto the next line, also.